assets-styleguide/js/styleguide.js

```diff
@@ -114,6 +121,17 @@ $('.sidenav').on('click', 'a', function(e) {
       } else if (window.location.hash !== newHash) {
         window.location.hash = newHash;
       }
+    },
+    done: function () {
```

Comment:
Yeah, I agree. I didn't notice too much of a delay since the animation is set to 200 ms so done fires super fast, and doing the focusing before breaks the animation. Maybe it's even better to just skip the animation entirely if the keyboard was used?